https://bugs.gentoo.org/890411
https://github.com/pygobject/pycairo/pull/318

From 9e78bd1acbaef6659dbe3076934ed779eac27a96 Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Sun, 23 Apr 2023 19:44:26 +0100
Subject: [PATCH] docs: Fix build failure due to extlinks with Sphinx 6

extlinks caption must be None or contain one %s.

Compatible with Sphinx >= 4.0.
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -48,10 +48,10 @@ def exec_module(path):
 ]
 
 extlinks = {
-    'fdobug': ('https://bugs.freedesktop.org/show_bug.cgi?id=%s', '#fdo-'),
-    'bug': ('https://github.com/pygobject/pycairo/issues/%s', '#'),
-    'pr': ('https://github.com/pygobject/pycairo/pull/%s', '#pr-'),
-    'user': ('https://github.com/%s', ''),
+    'fdobug': ('https://bugs.freedesktop.org/show_bug.cgi?id=%s', '#fdo-%s'),
+    'bug': ('https://github.com/pygobject/pycairo/issues/%s', '#%s'),
+    'pr': ('https://github.com/pygobject/pycairo/pull/%s', '#pr-%s'),
+    'user': ('https://github.com/%s', '%s'),
 }
 suppress_warnings = ["image.nonlocal_uri"]
 
